content/tutorial/03-analyze/README.md

```diff
@@ -0,0 +1,56 @@
+# Querying the Cow Subgraph
```

Comment:
I understand this shouldn't be reviewed, but why are these unrelated changes be added to the PR then?